manager/integration/tests/test_backing_image.py

```diff
@@ -694,3 +704,199 @@ def test_volume_wait_for_backing_image_condition(client): # NOQA
     volume_endpoint = get_volume_endpoint(volume2)
     vol2_cksum = get_device_checksum(volume_endpoint)
     assert vol1_cksum == vol2_cksum
+
+
+@pytest.mark.backing_image  # NOQA
+def test_backing_image_min_number_of_replicas(client): # NOQA
+    """
+    Test the backing image minNumberOfCopies
+
+    Given
+    - Create a BackingImage
+    - Update the minNumberOfCopies to 2
+
+    When
+    - The BackingImage is prepared and transferred to manager
+
+    Then
+    - There will be two ready file of the BackingIamge
+
+    Given
+    - Set the setting backing-image-cleanup-wait-interval to 1 minute
+
+    When
+    - Update the minNumberOfCopies to 1
+    - Wait for 1 minute
+
+    Then
+    - There will be one ready BackingImage file left.
+    """
+    bi_url = BACKING_IMAGE_RAW_URL
+    create_backing_image_with_matching_url(
+        client, BACKING_IMAGE_NAME, bi_url)
+
+    backing_image = client.by_id_backing_image(BACKING_IMAGE_NAME)
+    backing_image = backing_image.updateMinNumberOfCopies(minNumberOfCopies=2)
+
+    check_backing_image_disk_map_status(client, BACKING_IMAGE_NAME, 2, "ready")
+
+    update_setting(client, "backing-image-cleanup-wait-interval", "1")
+
+    backing_image = client.by_id_backing_image(BACKING_IMAGE_NAME)
+    backing_image = backing_image.updateMinNumberOfCopies(minNumberOfCopies=1)
+
+    time.sleep(60)
```

Comment:
We already check the number of copies and status by `check_backing_image_disk_map_status`. Do you need the 60-second sleep?

Reply:
you are right
check_backing_image_disk_map_status will poll the status for 150seconds
the cleanup time interval is set to 1min in this test case
we can rely on the check_backing_image_disk_map_status() to wait for the 1min